src/app/index.module.ts

```diff
@@ -193,23 +225,19 @@ angular.element(document).ready(() => {
         redirectUrl: keycloakSettings['che.keycloak.redirect_url.dashboard']
       };
       return keycloakInit(keycloakAuth.config, initOptions);
-    }).catch((error: any) => {
-      if (keycloakSettings['che.keycloak.js_adapter_url']) {
-        hasCertificateError = true;
-      }
-      return Promise.reject(error);
+    }).catch((errorMessage: string) => {
+      return Promise.reject(errorMessage);
```

Comment:
Do we really need it
```
.catch((errorMessage: string) => {
      return Promise.reject(errorMessage);
})
```
?
Maybe leave as is(without catching at all).
WDYT?

Reply:
yes, makes sense, thanks